tests/config/test_comm_config.cpp

```diff
@@ -22,7 +22,7 @@
 
 #include "communication/CommFactory.hpp"
 #include "KeyfileIOUtils.hpp"
-#include "config_file_parser.hpp"
+#include "../../util/include/config_file_parser.hpp"
```

Comment:
don't use relative paths in include, if needed, add the h file to the cmake file

Reply:
Done